src/Orchard.Web/Modules/Orchard.Redis/Caching/RedisCacheStorageProvider.cs

```diff
@@ -10,9 +10,13 @@
 
 namespace Orchard.Redis.Caching {
 
+    public interface IRedisCacheStorageProvider : ICacheStorageProvider {
```

Comment:
Could be a more general interface instead, like `ICacheStorageProviderWithKeyPrefix`

Reply:
What do you think @MpDzik?

Reply:
Fixed in 138b068e24a5980e740e078e823a6ce9cd1e7f9f.